static/js/filter.js

```diff
@@ -99,6 +99,14 @@ function message_matches_search_term(message, operator, operand) {
     case 'sender':
         return people.id_matches_email_operand(message.sender_id, operand);
 
+    case 'huddle-with':
+        var operands_ids = people.pm_with_operand_ids(operand);
+        if (!operands_ids) {
+            return false;
+        }
+
+        return true;
```

Comment:
Oh actually, after looking at the source of `pm_with_operand_ids`, you should do:
```js
return !!people.pm_with_operand_ids(operand);
```
or this if the linter complains:
```js
return Boolean(people.pm_with_operand_ids(operand));
```
to get the truthy/falsy value.